spotify_confidence/analysis/frequentist/confidence_computers/confidence_computer.py

```diff
@@ -385,12 +384,12 @@ def _compute_differences(
         if type(level_as_reference) is not bool:
             raise ValueError(f"level_as_reference must be either True or False, but is {level_as_reference}.")
         groupby = listify(groupby)
-        unique_levels = set([l[0] for l in levels] + [l[1] for l in levels])
+        unique_levels = set([level[0] for level in levels] + [level[1] for level in levels])
```

Comment:
Latest flake8 doesn't like `l` as variable name because it looks too similar to `I`. 🙂